internal/cmd/ske/kubeconfig/create/create_test.go

```diff
@@ -202,7 +212,7 @@ func TestParseInput(t *testing.T) {
 	}
 }
 
-func TestBuildRequest(t *testing.T) {
+func TestBuildRequestCreate(t *testing.T) {
```

Comment:
please add unit test for `BuildRequestLogin`

Reply:
`buildRequestLogin` only wraps the `apiClient.GetLoginKubeconfig` func.
I can remove the `buildRequestLogin` and use `apiClient.GetLoginKubeconfig` if you want.

Reply:
Again, mainly for uniformity with the rest of the codebase as we usually always test these `buildRequest` methods. But in this case it's really just a wrapper so you can skip this test if you want